python/fedml/api/modules/storage.py

```diff
@@ -438,10 +469,12 @@ def _get_storage_service(service):
     else:
         raise NotImplementedError(f"Service {service} not implemented")
 
-def _check_data_path(data_path):
-    if os.path.isdir(data_path) or os.path.isfile(data_path):
-        return True
-    return False
+def _get_data_type(data_path):
+    if os.path.isdir(data_path):
+        return "dir"
+    elif os.path.isfile(data_path):
+        return "file"
+    return "invalid"
```

Comment:
Let's use enums here

Reply:
I've added and test. Let me know if that was the expectation.